src/actions/CurrencyAction.js

```diff
@@ -1,6 +1,6 @@
 import { getEtherPerCurrency, toPromise } from '../utils';
 
-export const setCurrency = (currency, time = new Date().yyyymmdd(), dispatch) => {
+export const setCurrency = (currency, time = new Date().yyyymmdd(), dispatch, callback) => {
   let currencyFormat = `ETH-${currency}`;
   if (time === 'NOW') { time = new Date().yyyymmdd(); }
```

Comment:
why are you using 'NOW'? can't you just leave it null so time = new Date().yyyymmdd() will be correctly assigned?